CHANGELOG.md

```diff
@@ -2,6 +2,54 @@
 copyright: Copyright (c) K Team. All Rights Reserved.
 ---
 
+K Framework 6.0.0
+=================
+
+Features
+--------
+- Removed the Java backend. From now on, all symbolic execution will be handled by the
+  haskell backend and concrete execution by the llvm backend. Alongside that we also
+  removed the `kprove-legacy` and the `kbmc` tool. Bounded model checking capabilities
+  have been added to the pyk library.
+
+- Deprecated the `--directory` option. Use `--output-definition` instead.
```

Comment:
This needs to be updated because there are now different flags for kompile and krun